LEGO1/legoroi.cpp

```diff
@@ -1,16 +1,90 @@
 #include "legoroi.h"
 
+#include <string.h>
+
+// SIZE 0x14
+typedef struct {
+	const char* name;
+	MxS32 red;
+	MxS32 green;
+	MxS32 blue;
+	MxS32 unk10;
+} roi_color_alias;
```

Comment:
We should probably name this `RoiColorAlias` (or `ROIColorAlias`)  to stick to our naming conventions